### ISSUE TITLE
RFC: Possibility of adding `mock-builder` utility + introducing code owners

### ISSUE THREAD

**Issue by wischli** [open-web3-stack/open-runtime-module-library#959]
Hey there,

First off, a big shout-out and thank you to all the amazing folks behind this repository! You've been doing some fantastic work, and I've been a fan since day one. I can't thank you enough for the awesome stuff this place provides.

Over at Centrifuge, we've cooked up something we think you might like - it's called the `mock-builder` utility. It's all about making life easier when you're mocking up loosely connected components for Substrate unit tests. You can dive into all the nitty-gritty details in the [discussion thread](https://forum.polkadot.network/t/mock-builder-a-utility-to-create-mock-pallets/2955?u=williamfreude) over on the Polkadot Forum. Right now, it lives in [this repository](https://github.com/foss3/runtime-pallet-library) that's taking some cues from ORML, mostly because we don't have write access here.

We've been tossing around the idea of merging our `mock-builder` into ORML. If that's something you're down for, we'd be stoked to take on the code ownership responsibility. Our goal is to make sure it doesn't add to your maintenance load. Plus, we've got some big plans to revamp the `mock-builder` down the road, and that includes adding support for procedural macros. Without code ownership, that journey gets trickier for both of us.

One thing we noticed is that there's no `CODEOWNERS` file set up for this repo right now. Entirely giving up code ownership is a roadblock for us, but we still wanted to toss the idea out there. We were wondering whether introducing some code owners in general is something you have had in mind as well? Your thoughts on this would mean a lot to us. Thanks a bunch!

**Comment by xlc** [comment]
Yeah definitely happy to work with you to co-maintain mock-builder. Happy to grant you necessary permissions to allow you to manage it.

One question: Do you want it to be in this repo or a standalone repo under open-web3-stack github org?

A mono repo setup will make sense if other code in this repo also uses mock-builder and then we can have everything updated together. A separate repo works best if you are expecting frequent breaking changes.
